app/views/components/_cdx_select.html.haml

```diff
@@ -1,3 +1,3 @@
 - field_name = cdx_select[:form] ? ActionView::Helpers::Tags::TextField.new(cdx_select[:form].object_name, cdx_select[:name], cdx_select[:form], {}).send(:tag_name) : cdx_select[:name]
 - field_value = cdx_select.fetch(:value) { cdx_select[:form] ? cdx_select[:form].object.send(cdx_select[:name]) : nil }.to_s
-= react_component('CdxSelect', name: field_name, items: cdx_select[:items] || [], value: field_value, className: cdx_select[:class], multi: cdx_select[:multi])
+= react_component('CdxSelect', name: field_name, items: cdx_select[:items] || [], value: field_value, className: cdx_select[:class], multi: cdx_select[:multi], searchable: cdx_select[:searchable].nil? || cdx_select[:searchable])
```

Comment:
This will make the component `searchable` by default (ie, if `cdx_select[:searchable]` is nil, the `CdxSelect` has `searchable: true`. Are we sure that's what we want?